must-gather/collection-scripts/gather_pod_logs

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
We don't need to gather the pod logs again. This is done by the general mechanism of must-gather per namespace:
https://github.com/openshift/cluster-logging-operator/blob/master/must-gather/collection-scripts/gather#L9